substrate/client/sysinfo/src/sysinfo.rs

```diff
@@ -732,6 +788,13 @@ mod tests {
 		assert!(benchmark_cpu(DEFAULT_CPU_EXECUTION_LIMIT) > Throughput::from_mibs(0.0));
 	}
 
+	#[test]
+	fn test_benchmark_parallel_cpu() {
+		assert!(
+			benchmark_cpu_parallelism(DEFAULT_CPU_EXECUTION_LIMIT) > Throughput::from_mibs(0.0)
```

Comment:
This needs to be updated wrt reference hardware tput.

Reply:
I'm not sure about that, as far I can tell the only purpose of this tests is to run `benchmark_*` make it  is producing results, otherwise the compiling machine would have to always match the reference hardware spec.

The reference hardware check is done when each of the validator starts it here:  https://github.com/paritytech/polkadot-sdk/pull/5127/files/213953c03591d6afe376334ec7ad04172289ccbb#diff-7ddbac9b9ba846f3111f1248a02e36d03206fe64524be6714bbb83a1825dcf6dR1084